src/components/ftbot/ForceEntryForm.vue

```diff
@@ -29,10 +29,16 @@
         <b-form-group label="Pair" label-for="pair-input" invalid-feedback="Pair is required">
           <b-form-input
             id="pair-input"
-            v-model="pair"
+            v-model="selectedPair"
+            list="pair-input-whitelist"
             required
             @keydown.enter.native="handleEntry"
+            @focus="inputSelect"
           ></b-form-input>
+          <b-form-datalist
```

Comment:
I don't really like the feel of the dataList. It's odd and "moves around" (and it's too small, really).

what we could use is the `v-select` we use for pair selection at the top of the regular chart - assuming that also allows selection of "non-available" pairs (via forcebuy, we don't restrict to the pair whitelist).

Reply:
Oh I think it uses the browser's own autocomplete thing and isn't a consistent experience then. I'll remove it.

From what I remember v-select didn't really allow it out of the box at least though.